lib/spree/shipment_notice.rb

```diff
@@ -29,6 +44,7 @@ def update
       shipment.update_attribute(:tracking, tracking)
 
       unless shipment.shipped?
+        shipment.reload.ready! unless shipment.ready?
```

Comment:
We probably won't be able to `ready!` the shipment if `paid` is being called after `locate` no?